src/utils/helpers/date-and-time-formatter.ts

```diff
@@ -0,0 +1,37 @@
+const getDate = (dateParam: Date | string): string => {
+  if (!dateParam) {
+    console.error('getDate() failed');
+    return '';
+  }
+
+  const date = new Date(dateParam);
+  const nowDate = Date.now();
+  const diff = nowDate - date.getTime();
```

Comment:
`Date.now()` er såppas kort og forståelig at jeg tenker det ikke er nødvendig med en ekstra const.
```suggestion
  const diff = Date.now() - date.getTime();
```